### PR TITLE
feat: create job experiments on createJob logics

### PR DESCRIPTION
**Pull request by GuyAb**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |

Related issues: #XXX , #XXX ...
Closes #XXX , #XXX ...

Further  information:

### CODE REVIEW

File: src/chaos-experiments/models/database/databaseConnector.js [hunk 1]
```diff
@@ -44,6 +45,10 @@ async function getChaosExperimentByName(name, contextId) {
     return databaseConnector.getChaosExperimentByName(name, contextId);
 }
 
+async function getChaosExperimentsByIds(experimentIds, contextId) {
```
Comment (kerenfi) [inline src/chaos-experiments/models/database/databaseConnector.js:48]: checkout the whole file - we have some dups

File: src/jobs/models/jobExperimentsHandler.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    if (!jobExperiments) {
+        return;
+    }
+    try {
+        const baseTimestamp = Date.now();
+        const experimentIds = jobExperiments.map(experiment => experiment.experiment_id);
+        const experiments = await chaosExperimentsDbConnector.getChaosExperimentsByIds(experimentIds);
+        for (const experimentRequest of jobExperiments) {
+            try {
+                const experiment = experiments.find(e => e.id === experimentRequest.experiment_id);
```
Comment (kerenfi) [inline src/jobs/models/jobExperimentsHandler.js:22]: maybe  just for readability would be good to separate the content inside the for loop to another function
setSingleJobExperiment - don't like the catch inside the cate

Comment (GuyAb) [reply src/jobs/models/jobExperimentsHandler.js]: I did it and changed the logic to be async (with promise all and map) - plz take a look again

Comment (kerenfi) [reply src/jobs/models/jobExperimentsHandler.js]: 🍻

**Pull request merged**